pilot/pkg/networking/util/util.go

```diff
@@ -257,6 +257,12 @@ func IsIstioVersionGE116(version *model.IstioVersion) bool {
 		version.Compare(&model.IstioVersion{Major: 1, Minor: 16, Patch: -1}) >= 0
 }
 
+// IsIstioVersionGE117 checks whether the given Istio version is greater than or equals 1.17.
+func IsIstioVersionGE117(version *model.IstioVersion) bool {
+	return version == nil ||
```

Comment:
I don't think version==nil is greater ?

Reply:
This is a copy paste from above. Not sure why it checks for nil.